### PR TITLE
Fix some stylelint warnings

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: marco <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/ConversationIcon.vue [hunk 1]
```diff
@@ -155,7 +154,7 @@ $icon-size: 44px;
 		width: $icon-size;
 		height: $icon-size;
 		line-height: $icon-size;
-		font-size: math.div($icon-size, 2);
+		font-size: $icon-size / 2;
```
[the remark this answers is not in the document]
Comment (marcoambrosini) [reply src/components/ConversationIcon.vue:157]: Hmm, why do we get the warning in these two files only? We use this / everywhere outside calc()

**Comment by marcoambrosini** [comment]
Anyway, updated!

**Review by nickvergessen — APPROVED** [review]

**Comment by marcoambrosini** [comment]
Updated so that we allow only the rules that we need

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**